axonserver/src/main/java/io/axoniq/axonserver/localstorage/LocalEventStore.java

```diff
@@ -92,13 +92,13 @@
     private final EventDecorator eventDecorator;
     private volatile boolean running;
     @Value("${axoniq.axonserver.query.limit:200}")
-    private long defaultLimit = 200;
+    private final long defaultLimit = 200;
```

Comment:
Just for my understanding, if we make these fields final does the @Value annotation still work?

Reply:
I don't think. Intellij cleanup feature remove it during the commit.